mlflow/server/js/src/experiment-tracking/components/ExperimentView.js

```diff
@@ -396,11 +396,15 @@ export class ExperimentView extends Component {
   }
   // END-EDGE
 
+  getUrl() {
```

Comment:
@harupy 
This works well even if in iframe.
I commit small change but if you've already fixed, please close this pull request.
Thank you for teaching and maintaining very good mlflow UI.

Reply:
@dorakemon 

> I commit small change but if you've already fixed, please close this pull request.

We've fixed the issue in our private repository and it'll be shipped in the next MLflow release. Again, thank you for finding this issue. We really appreciate it.